update readme (#2)
opengisch/qgis-plugin-ci: README.md

```
deploy:

  - provider: script
    script: qgis-plugin-ci release ${TRAVIS_TAG} --github-token ${GH_TOKEN} --osgeo-username ${OSGEO_USERNAME} --osgeo-password {OSGEO_PASSWORD}
    on:
      tags: true
```

> This assumes that you have an existing GitHub release. If you want, Travis can create the release itself. In the `deploy` section, you must add another `provider` before the script `qgis-plugin-ci`:

Reviewer: ```suggestion
This assumes that you have an existing GitHub release. 
Alternatively, Travis can create the release by using the `releases`provider:
```

```
deploy:

  - provider: script
    script: qgis-plugin-ci release ${TRAVIS_TAG} --github-token ${GH_TOKEN} --osgeo-username ${OSGEO_USERNAME} --osgeo-password {OSGEO_PASSWORD}
    on:
      tags: true
```

This assumes that you have an existing GitHub release. 

> Alternatively, Travis can create the release by adding  a `releases`provider before the `script` provider:

Author: ```suggestion
Alternatively, Travis can create the release by adding a `releases` provider before the `script` provider:
```
